lite/doc.go

```diff
@@ -56,35 +54,32 @@ greater).
 DynamicVerifier - this Verifier implements an auto-update and persistence
 strategy to verify any SignedHeader of the blockchain.
 
-## Provider and PersistentProvider
+Provider and PersistentProvider
 
 A Provider allows us to store and retrieve the FullCommits.
 
-```go
-type Provider interface {
+ type Provider interface {
```

Comment:
code blocks need to be indented (example https://github.com/googleapis/google-cloud-go/blob/master/datastore/doc.go#L57-L59)

Reply:
Its at least one extra space to the other comment content, which should be rendered as a code example:

<img width="739" alt="image" src="https://user-images.githubusercontent.com/43315617/51663004-d2577780-1fb5-11e9-9065-820fc022da8c.png">

I now saw  in crypto/doc.go that you have 4 spaces extra. I've applied that to the two examples  👍